examples/ios/swift/AppClip/AppClipApp.swift

```diff
@@ -0,0 +1,41 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2020 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import SwiftUI
+import RealmSwift
+
+@main
+struct AppClipApp: SwiftUI.App {
+    var body: some Scene {
+        WindowGroup {
+            // This is ContentView.swift shared from AppClipParent
+            ContentView(objects: demoObjects().list)
+        }
+    }
+
+    private func demoObjects() -> DemoObjects {
+        let config = Realm.Configuration(fileURL: FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: Constants.groupId)!.appendingPathComponent("default.realm"))
```

Comment:
Why not make `FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: Constants.groupId)!.appendingPathComponent("default.realm")` as part of a constant struct?

Reply:
Ah, I didn't notice that. That's be fine. But I guess since this is the primary important line in the example, having it written in line for both apps wouldn't hurt. Especially if someone were just viewing either the AppClip or ParentApp on Github, instead of an editor.